src/pages/[id].page.jsx

```diff
@@ -24,24 +24,42 @@ export default function Details({ details }) {
 
   return (
     <Layout pageTitle={componentName}>
-      <div className="details-page">
-        <DetailsBanner name={componentName} image={image} />
+      <div className="details-page__header">
+        <div className="details-page__primary">
+          <DetailsBanner name={componentName} image={image} />
+        </div>
+      </div>
+      <div className="details-page__primary">
         <Definition>
           <p>{definition}</p>
```

Comment:
I checked the `<p>` text on figma and it actually seemed that the current text is too large at 20px, and should be 16px instead. I think this is what you meant since it's what's shown in your screenshots, so I reduced all the font sizes in the mobile view.